util/logtools/splitter.ts

```diff
@@ -67,6 +105,22 @@ export default class Splitter {
       const combatantId = splitLine[idIdx]?.toUpperCase();
       if (combatantId !== undefined)
         delete this.addedCombatants[combatantId];
+    } else if (typeField === '262') {
+      // if we receive a 262 line after the 01 line, this means a new zone change is occurring
+      // so reset rsvLines/rsvSubstitutionMap and recollect
+      if (this.rsvLinesReceived) {
+        this.rsvLinesReceived = false;
+        this.rsvLines = {};
+        this.rsvSubstitutionMap = {};
+      }
+      const idIdx = 4;
+      const valueIdx = 5;
+      const rsvId = splitLine[idIdx];
+      const rsvValue = splitLine[valueIdx];
+      if (rsvId !== undefined && rsvValue !== undefined) {
+        this.rsvLines[rsvId] = line;
+        this.rsvSubstitutionMap[rsvId] = rsvValue;
```

Comment:
Do you feel like there's any need to split statuses and actions? Proooobably we're ok in that they are wildly apart from each other and likely won't ever collide, but they are separate namespaces of ids.

Could you at least leave a TODO here about this?

Reply:
> Do you feel like there's any need to split statuses and actions? Proooobably we're ok in that they are wildly apart from each other and likely won't ever collide, but they are separate namespaces of ids.
> 
> Could you at least leave a TODO here about this?

Yeah, proooobably okay, and I will leave a comment.  Although more to the point, I'm not sure we get the namespace from the 262 line anyway.  There's the (commented-out) `unknown0` (3) field on the line that I suspected might have some relation to namespace, but it doesn't match up with the exd table ids so unless we make some assumptions and build a mapping, I'm not sure how else to validate that.